src/components/pages/Categories/Update.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
+    <app-category-update
+      class="category-update"
+      :category-id="categoryId"
+      :category-name="categoryName"
+      :access="access"
+      :loading="loading"
+      :update-category="updateCategory"
+      :done-message="doneMessage"
+      :error-message="errorMessage"
+      @edit-name="editName"
+      @handle-submit="handleSubmit"
+    />
+  </div>
+</template>
+
+<script>
+import { CategoryUpdate } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryUpdate: CategoryUpdate,
+  },
+  computed: {
+    categoryName() {
+      return this.$store.state.categories.updateCategory.name;
+    },
+    categoryId() {
+      const id = parseInt(this.$route.params.id, 10);
+      return id;
+    },
```

Comment:
[imo]
そのままreturnしても良いかなと思いました！

```suggestion
    categoryId() {
      return parseInt(this.$route.params.id, 10)
    },
```

Reply:
=以下の内容をそのままreturnしました。